client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/survey/survey.tsx

```diff
@@ -0,0 +1,40 @@
+import { useLocale } from '@automattic/i18n-utils';
+import { useState } from 'react';
+import Banner from 'calypso/components/banner';
+import './survey.scss';
+
+const Survey = ( { userHasEngaged }: { userHasEngaged: boolean } ) => {
+	const locale = useLocale();
+	const [ dismissed, setDismissed ] = useState( false );
+
+	if ( 'en' !== locale || dismissed || ! userHasEngaged ) {
```

Comment:
I think we're able to show the survey for the local starting with `en-` as well. WDYT?

Reply:
We are only targeting users with the English interface language from this list. I cannot find other English `en-` locales there.

<img width="1040" alt="Screenshot 2566-06-06 at 09 43 49" src="https://github.com/Automattic/wp-calypso/assets/1881481/244b344c-486a-4d03-8ac7-945e59d48fc9">

Reply:
Nevermind, let's show it for all the English variations.